bgpd/bgp_errors.h

```diff
@@ -100,6 +100,9 @@ enum bgp_log_refs {
 	EC_BGP_CAPABILITY_UNKNOWN,
 	EC_BGP_INVALID_NEXTHOP_LENGTH,
 	EC_BGP_DOPPELGANGER_CONFIG,
+	EC_BGP_FIB_INSTALL_ERROR,
+	EC_BGP_INVALID_NODE,
+	EC_BGP_INVALID_ROUTE,
```

Comment:
You need to update the corresponding `bgp_errors.c` file with a struct for each of these errors.